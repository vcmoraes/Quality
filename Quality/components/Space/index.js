import React from 'react'
import styled from 'styled-components'

const Space = styled.View``

export default props => <Space {...props} />
