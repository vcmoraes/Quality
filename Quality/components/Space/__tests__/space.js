import React from 'react'
import Space from '../index'
import renderer from 'react-test-renderer'

describe('Components - Space', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Space />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
