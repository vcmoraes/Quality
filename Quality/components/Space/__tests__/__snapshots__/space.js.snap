// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components - Space renders correctly 1`] = `
<View
  style={
    Array [
      Object {},
      undefined,
    ]
  }
/>
`;
